tests/browser/rollup.js

```diff
@@ -0,0 +1,16 @@
+const { join } = require('path')
+const { nodeResolve } = require('@rollup/plugin-node-resolve')
+const json = require('@rollup/plugin-json')
+
+module.exports = {
+  input: join(__dirname, 'spec.js'),
+  output: {
+    file: join(__dirname, 'dist.js'),
+    format: 'iife',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
    format: "iife",
```